tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1286,8 +1286,13 @@ public function updateSpaceName(
 		string $spaceName,
 		string $newName
 	): void {
-		$space = $this->getSpaceByName($user, $spaceName);
-		$spaceId = $space["id"];
+		if ($spaceName === "unknown") {
+			// check sending invalid data
+			$spaceId = "39c49dd3-1f24-4687-97d1-42df43f71713";
```

Comment:
instead of providing explicit  space id cant we create a function that generated random space id for unknown space? we have a function in core `public static function generateUUIDv4()` that generated random UUIDv4.

Reply:
please don't do random generated values in the regular API tests, random values will lead to random failures and they will be a nightmare to debug.
If we want to test with really random data that would be an other type of test suite

Reply:
@ScharfViktor

Reply:
returned `$spaceId = "39c49dd3-1f24-4687-97d1-42df43f71713";`